libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/AudioCapabilities.java

```diff
@@ -95,6 +96,19 @@ public static AudioCapabilities getCapabilities(Context context) {
     return getCapabilities(context, intent);
   }
 
+  private static AudioCapabilities getExternalSurroundCapabilities(@Nullable Intent intent) {
+    int[] supportedEncodings = EXTERNAL_SURROUND_SOUND_CAPABILITIES;
+
+    if (Util.SDK_INT >= 29) {
+      // Check if DTS Encodings are supported via Direct Playback.
+      int[] dtsTypes = {AudioFormat.ENCODING_DTS, AudioFormat.ENCODING_DTS_HD};
```

Comment:
nit: Better to rename `dtsTypes` to `dtsEncodings`?

Reply:
@tianyif Done. Please see 8072d63acd4cbbe6666c68f5865bfb439fbeb23c Thanks.